docs/guide/features.md

```diff
@@ -296,6 +296,15 @@ Note that:
 - The glob patterns are treated like import specifiers: they must be either relative (start with `./`) or absolute (start with `/`, resolved relative to project root).
 - The glob matching is done via `fast-glob` - check out its documentation for [supported glob patterns](https://github.com/mrmlnc/fast-glob#pattern-syntax).
 
+You should also be aware that glob imports can not accept variables, You must hard code the imports. The most elegent solution to this is a switch statement, like so:
+
+```js
+switch (url) {
+  case "posts":  location = Object.entries(import.meta.globEager("/posts/**/*.md"));
+  case "assets": location = import.meta.globEager("assets/**/*.{png,jpg}");
+}
+```
```

Comment:
```suggestion
You should also be aware that glob imports can not accept variables. You must hard code the imports. The most elegant solution to this is a switch statement, like so:

```js
switch (url) {
  case "posts": location = Object.entries(import.meta.globEager("/posts/**/*.md"));
  case "assets": location = import.meta.globEager("assets/**/*.{png,jpg}");
}
```
```

Reply:
Thanks for this addition. I think we could replace "The most elegant solution to this is a..." with "A possible way to achieve this is using...". One problem I see with the switch is that it could give the impression that sometimes only some imports are triggered.

Defining the glob imports before having to do the switch is more clear IMO in this regard:
```js
const location = {
  posts: Object.entries(import.meta.globEager("/posts/**/*.md")),
  assets: import.meta.globEager("assets/**/*.{png,jpg}")
}
// location[url]
```

Reply:
sounds good 👍🏻